aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/css/materialize.css

```diff
@@ -376,7 +376,6 @@ form p:last-child {
   height: 15px;
   background-color: #818181;
   border-radius: 15px;
-  margin-right: 10px;
```

Comment:
can this be merged with the `margin` below?

Reply:
Acually i want to change  
' margin-top: 0px;
    margin-right: 10px;
    margin-bottom: 0px;
    margin-left: 16px; '

but it is already overrid to 16px so i was not sure so i set like that. i can change? should i change?